docs/4.1/admin-guide.md

```diff
@@ -2328,17 +2296,13 @@ When running multiple binaries of Teleport within a cluster (nodes, proxies,
 clients, etc), the following rules apply:
 
 * Patch versions are always compatible, for example any 4.0.1 component will
-
   work with any 4.0.3 component.
 
 * Other versions are always compatible with their **previous** release. This
-
   means you must not attempt to upgrade from 3.3 straight to 3.5. You must
   upgrade to 3.4 first.
 
-* Teleport clients ( [ `tsh` ](../cli-docs/#tsh) for users and
-
-  [ `tctl` ](../cli-docs/#tctl) for admins) may not be compatible
+* Teleport clients ( [ `tsh` ](../cli-docs/#tsh) for users and  [ `tctl` ](../cli-docs/#tctl) for admins) may   not be compatible
```

Comment:
`may not be` (lots of extra spaces)

'and [`tctl`]' (extra space beforehand)

Also there's a space after the bracket at one end but not a matching one at the other.